client/my-sites/themes/theme-showcase.jsx

```diff
@@ -162,9 +162,7 @@ class ThemeShowcase extends Component {
 			return null;
 		}
 
-		const shouldShowMyThemesFilter =
-			( this.props.isJetpackSite && ! this.props.isAtomicSite ) ||
-			( this.props.isAtomicSite && this.props.siteCanInstallThemes );
+		const shouldShowMyThemesFilter = config.isEnabled( 'themes/tiers' ) && !! this.props.isLoggedIn;
```

Comment:
We can remove the feature check. We have an issue with adding the my themes tab to simple sites.

Reply:
Got it. I decided to handle the My themes logic in separate PR because it needs more work on the back-end. :)